### PR TITLE
doc: New Python Bottle integration

### PR DESCRIPTION
**Pull request by shenek**
related to https://github.com/getsentry/sentry-python/pull/300#issuecomment-477344421 which is being supervised by @untitaker

### CODE REVIEW

File: src/_data/platforms.yml [hunk 1]
```diff
@@ -267,8 +267,7 @@
       doc_link: /clients/python/integrations/bottle/
```
Comment (untitaker) [inline src/_data/platforms.yml]: This is still referring to the old Raven SDK while your updated lines refer to the new client.

It should be `/platforms/python/bottle/`

File: src/collections/_documentation/platforms/python/bottle.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+  ```python
+  @app.route("/myurl/<foo>")
+  def myendpoint():
+      return "ok"
+  ```
+
+  In the above code, you would set the transaction to:
+
+  * `/myurl/<foo>` if you set `transaction_style="url"`. This matches the behavior of the old Raven SDK.
```
Comment (untitaker) [inline src/collections/_documentation/platforms/python/bottle.md]: Is this true though? I just took a quick look at the old Bottle integration in Raven, and it seems like we never set the transaction to anything.

File: src/collections/_documentation/platforms/python/bottle.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  * `myendpoint` if you set `transaction_style="endpoint"`
+
+  The default is `"endpoint"`.
+
+## User Feedback
+
+You can use the user feedback feature with this integration.  For more information see [User Feedback]({% link _documentation/enriching-error-data/user-feedback.md %}?platform=bottle).
```
Comment (untitaker) [inline src/collections/_documentation/platforms/python/bottle.md]: This link will not work. If you want to make an example for user feedback, there's the file `_site/documentation/enriching-error-data/user-feedback-example/flask.md` that you'd need to copy. But we didn't do this for all web frameworks either.

**Comment by shenek** [comment]
> Looks good so far! Did you check locally how this looks?

Yes. The installation took more time than actual doc writing. Mostly caused by a slow train connection.


sentry-bottle-icons

sentry-bottle-page

File: src/_data/platforms.yml (hunk 1 above)
Comment (shenek) [reply src/_data/platforms.yml]: Fixed. Thanks for noticing.

File: src/collections/_documentation/platforms/python/bottle.md (hunk 2 above)
Comment (shenek) [reply src/collections/_documentation/platforms/python/bottle.md]: You're right. This part was originally taken from flask integration and I forgot to check whether it is also valid.

File: src/collections/_documentation/platforms/python/bottle.md (hunk 3 above)
Comment (shenek) [reply src/collections/_documentation/platforms/python/bottle.md]: Ok, I removed it for now.

File: src/collections/_documentation/platforms/python/bottle.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+sidebar_order: 5
+---
+
+{% version_added 0.7.9 %}
+
+
+<!-- WIZARD -->
+*Import name: `sentry_sdk.integrations.bottle.BottleIntegration`*
+
+The Bottle integration adds support for the [Bottle Web Framework](https://bottlepy.org/).
```
Comment (untitaker) [inline src/collections/_documentation/platforms/python/bottle.md]: Could you specify here which versions of bottle the integration supports? We should have a sentence like that for some of the existing integrations.

File: src/collections/_documentation/platforms/python/bottle.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
+---
```
Comment (untitaker) [inline src/collections/_documentation/platforms/python/bottle.md]: I almost overlooked it, you need to add a link to this page in `index.md` (in the same folder) for completeness

Comment (shenek) [reply src/collections/_documentation/platforms/python/bottle.md]: Added a link. Thanks for noticing.

File: src/collections/_documentation/platforms/python/bottle.md (hunk 4 above)
Comment (shenek) [reply src/collections/_documentation/platforms/python/bottle.md]: A note was added. I also discovered that it doens't work well with the development version of bottle (0.13). Probably I need to redo some of the patching.

**Comment by untitaker** [comment]
Great work, thanks! 0.7.9 is released and the docs deploy on merge.

**Pull request merged**